release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -651,6 +651,15 @@ You must still use the `performance-addon-operator-must-gather` image when runni
 [id="ocp-4-11-insights-operator"]
 === Insights Operator
 
+==== Insights Operator data collection enhancements
+In OpenShift Container Platform 4.11, the Insights Operator collects the following additional information:
```

Comment:
This is a great opportunity to use the `{product-title} {product-version}` [attributes](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#product-name-and-version).